src/App.js

```diff
@@ -1,24 +1,74 @@
-import logo from './logo.svg';
+import React, { useEffect } from 'react';
 import './App.css';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+// Import the necessary components
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import SignInPage from './pages/signin/SignIn';
+import Layout from './components/layout/Layout';
+import Main from './routes/Home';
+import Delete from './routes/Delete';
+import AddNew from './routes/AddNew';
+import MyReservations from './routes/MyReservations';
+import Reserve from './routes/Reserve';
+import ItemDetail from './routes/ItemDetail';
+import { usersSelector } from './redux/store';
+import createUser from './redux/user/thunk';
+import LoadingScreen from './components/conditions/LoadingScreen';
 
 function App() {
```

Comment:
- [ ] Kindly, remove these `console.log` debugging statements, in order to improve the repo's cleanliness. 👍 

![image](https://github.com/Salimer/Final-Capstone-Front-End/assets/57421823/c0b6ddbc-b883-4888-af5a-46b4360f944c)